packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/DynamicTable/TableRows/tests/index.test.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { IntlProvider } from 'react-intl';
+import { createMemoryHistory } from 'history';
+import { fireEvent, render } from '@testing-library/react';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import TableRows from '..';
+
+const history = createMemoryHistory();
+
+const headers = [
+  {
+    name: 'action',
+    key: 'action',
+    metadatas: { label: 'Action', sortable: true },
+  },
+  {
+    key: 'date',
+    name: 'date',
+    metadatas: { label: 'Date', sortable: true },
+  },
+  {
+    key: 'user',
+    name: 'user',
+    metadatas: { label: 'User', sortable: true },
+  },
+];
+
+const rows = [
+  {
+    id: 1,
+    action: 'Update',
+    date: '2022-11-14T23:04:00.000Z',
+    user: 'John Doe',
+  },
+  {
+    id: 2,
+    action: 'Create',
+    date: '2022-11-04T18:24:00.000Z',
+    user: 'Kai Doe',
+  },
+];
+
+const onModalToggle = jest.fn();
+
+// eslint-disable-next-line react/prop-types
+const App = (
+  <ThemeProvider theme={lightTheme}>
+    <IntlProvider locale="en" messages={{}} defaultLocale="en" textComponent="span">
+      <Router history={history}>
+        <TableRows headers={headers} rows={rows} onModalToggle={onModalToggle} />
+      </Router>
+    </IntlProvider>
+  </ThemeProvider>
+);
+
+describe('ADMIN | Pages | AUDIT LOGS | ListView | Dynamic Table | Table Rows', () => {
+  it('should show the row data', () => {
+    const { getByText } = render(App);
```

Comment:
It's [recommended](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen) to use getByText from `screen` rather than from render

Reply:
noted, but I wanted to avoid screen.getByText to use every time and just destructure once and use it.